1. CWE Base Mod/Cold War Era Mod (CWE)/common/goods/00_goods.txt

```diff
@@ -610,36 +610,34 @@ fine_art = {
 	traded_quantity = 5
 }
 
-#luxury_clothes = {
-#	texture = "gfx/interface/icons/goods_icons/luxury_clothes.dds"
-#	cost = 60
-#	category = luxury
-#	
-#	obsession_chance = 1.0
-#	prestige_factor = 10
-#	convoy_cost_multiplier = 0.75
-#}
-
-#luxury_furniture = {
-#	texture = "gfx/interface/icons/goods_icons/luxury_furniture.dds"
-#	cost = 60
-#	category = luxury
-#	
-#	obsession_chance = 1.0
-#	prestige_factor = 10
-#	convoy_cost_multiplier = 0.75
-#}
-
-#luxury_automobiles = { #Produced by Germany
-#	texture = "gfx/interface/icons/goods_icons/luxury_automobiles.dds"
-#	cost = 100
-#	category = luxury
-#	
-#	obsession_chance = 2.0
-#	prestige_factor = 10
-#	traded_quantity = 8
-#}
+luxury_clothes = {
+	texture = "gfx/interface/icons/goods_icons/luxury_clothes.dds"
+	cost = 60
+	category = luxury
+	
+	obsession_chance = 1.0
+	prestige_factor = 10
+	convoy_cost_multiplier = 0.75
+}
 
+luxury_furniture = {
+	texture = "gfx/interface/icons/goods_icons/luxury_furniture.dds"
+	cost = 60
+	category = luxury
+	
+	obsession_chance = 1.0
+	prestige_factor = 10
+	convoy_cost_multiplier = 0.75
+}
+luxury_automobiles = { #Produced by Germany
```

Comment:
I’m planning to replace this with advanced automobiles to represent electric cars